Dockerfile

```
@@ -1,4 +1,4 @@
-FROM alpine:3.12
+FROM alpine:3.13.0
```

Probably good to keep this as `major.minor`, so that patch releases would still be picked up

```suggestion
FROM alpine:3.13
```